Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -77,7 +85,10 @@ public partial class GazeManager : Singleton<GazeManager>
 
         private void Awake()
         {
-            GazeStabilization = gameObject.GetComponent<GazeStabilizer>();
+            if (UseBuiltInGazeStabilization)
+            {
+                GazeStabilization = gameObject.AddComponent<GazeStabilizer>();
```

Comment:
What if my object already contains this component? Should we check for existence before adding another?